packages/gateway/src/features/formDraft/schema.graphql

```diff
@@ -58,16 +58,22 @@ input QuestionInput {
 
 input FormDraftInput {
   questions: [QuestionInput]
-  deleted: [String]
   event: String!
   status: String!
   comment: String
 }
 
+input FormDraftStatusModify {
+  event: String!
+  status: String!
+}
+
 type Query {
   getFormDraft: [FormDraft!]
 }
 
 type Mutation {
   createOrUpdateFormDraft(formDraft: FormDraftInput!): FormDraft
+  modifyDraftStatus(formDraft: FormDraftStatusModify!): FormDraft
+  deleteFormDraft(formDraft: FormDraftStatusModify!): FormDraft
```

Comment:
We really don't need two separate Mutations for this. As we can already see the inputs are also same